htlcswitch/link.go

```diff
@@ -1720,8 +1720,11 @@ func (l *channelLink) handleUpstreamMsg(msg lnwire.Message) {
 		// Since we just revoked our commitment, we may have a new set
 		// of HTLC's on our commitment, so we'll send them over our
 		// HTLC update channel so any callers can be notified.
+		newHTLCS := map[contractcourt.HtlcSetKey][]channeldb.HTLC{
```

Comment:
defining a new type, e.g. `contractcourt.HtlcSets` would help cut down on the verbosity here